pipelines/build/common/build_base_file.groovy

```diff
@@ -446,15 +446,22 @@ class Builder implements Serializable {
     */
     def getJobFolder() {
         def parentDir = currentBuild.fullProjectName.substring(0, currentBuild.fullProjectName.lastIndexOf("/"))
+
+        // weekly-release builds use parent nightly jobs
+        if (parentDir.endsWith("/weekly-release")) {
+            parentDir = parentDir.substring(0, parentDir.lastIndexOf("/weekly-release"))
+        }
+
         return parentDir + "/jobs/" + javaToBuild
     }
 
     /*
     Ensures that we don't release multiple variants at the same time
+    Unless this is the weekend weekly-release build
     */
     def checkConfigIsSane(Map<String, IndividualBuildConfig> jobConfigurations) {
 
-        if (release) {
+        if (release && !currentBuild.fullProjectName.contains("/weekly-release")) {
```

Comment:
Would it be simpler to test for the positive 'nightly' case?

Reply:
the nightly project is a sub-path of the weekly-release, so it's actually easier this way.